shell/platform/darwin/ios/framework/Source/FlutterRestorationPlugin.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SHELL_PLATFORM_IOS_FRAMEWORK_SOURCE_FLUTTERRESTORATIONPLUGIN_H_
+#define SHELL_PLATFORM_IOS_FRAMEWORK_SOURCE_FLUTTERRESTORATIONPLUGIN_H_
+
+#import <UIKit/UIKit.h>
+
+#include "flutter/fml/memory/weak_ptr.h"
```

Comment:
Doesn't look like you're using weak_ptr here (anymore).

Reply:
Removed import.